### PR TITLE
Sync definition of all imported actuator types with the latest changes

### PR DESCRIPTION
**Pull request by ingvagabund**
By running:
```sh
$ dep ensure -update sigs.k8s.io/cluster-api-provider-azure \
                     sigs.k8s.io/cluster-api-provider-aws \
                     sigs.k8s.io/cluster-api-provider-openstack \
                     github.com/openshift/cluster-api-provider-libvirt \
                     github.com/openshift/cloud-credential-operator \
                     github.com/openshift/cluster-api
```

### CODE REVIEW

**Comment by ingvagabund** [comment]
FYI @zeenix @tomassedovic @enxebre @michaelgugino @awesomenix

**Comment by ingvagabund** [comment]
https://github.com/openshift/cloud-credential-operator/pull/75 will resolves the unneeded import of `sigs.k8s.io/controller-runtime/pkg/client`.

File: pkg/asset/installconfig/aws/permissions.go [hunk 1]
```diff
@@ -193,14 +192,14 @@ func ValidateCreds(ssn *session.Session) error {
 		return errors.Wrap(err, "getting creds from session")
 	}
 
-	client, err := ccaws.NewClient([]byte(creds.AccessKeyID), []byte(creds.SecretAccessKey))
+	client, err := ccaws.NewClient([]byte(creds.AccessKeyID), []byte(creds.SecretAccessKey), "openshift-installer")
```
Comment (abhinavdahiya) [inline pkg/asset/installconfig/aws/permissions.go]: I would like to pass something like https://github.com/openshift/installer/blob/e1e3c787398632a227df9b0598095536cf2e3547/pkg/asset/installconfig/aws/aws.go#L108 to the client..

**Comment by ingvagabund** [comment]
`ci/prow/e2e-aws-scaleup-rhel7` timeouts waiting for nodes to get ready.

Based on https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/1872/pull-ci-openshift-installer-master-e2e-aws-scaleup-rhel7/300/artifacts/e2e-aws-scaleup-rhel7/nodes.json, there are 6 nodes listed. All nodes were created, all nodes joined the cluster, all nodes are reporting readiness.

Not sure if `ci/prow/e2e-aws-scaleup-rhel7` is using the actuators at all.

**Comment by patrickdillon** [comment]
> Not sure if ci/prow/e2e-aws-scaleup-rhel7 is using the actuators at all.

It does not. Scaleup test is currently broken. The test is non-blocking.

**Pull request merged**